client/my-sites/plans-single-products/index.jsx

```diff
@@ -2,33 +2,103 @@
  * External dependencies
  */
 import React from 'react';
-import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import { isEnabled } from 'config';
 import Product from './product';
+import { getCurrentUserCurrencyCode } from 'state/current-user/selectors';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const PlansSingleProducts = ( { translate } ) => {
+const PlansSingleProducts = props => {
+	const {
+		billingTimeFrame,
+		currencyCode,
+		isPlaceholder,
+		onProductSelect,
+		productProperties,
+	} = props;
 	const displayBackup = isEnabled( 'plans/jetpack-backup' );
 	const displayScan = isEnabled( 'plans/jetpack-scan' );
 
-	if ( ! displayBackup && ! displayScan ) {
-		return null;
-	}
-
 	return (
 		<div className="plans-single-products">
-			{ displayScan && <Product heading={ translate( 'Jetpack Scan' ) } /> }
-			{ displayBackup && <Product heading={ translate( 'Jetpack Backup' ) } /> }
+			{ displayScan && (
```

Comment:
I don't see any logic that cares about the current plan. What if the user already has that plan?

Reply:
We didn't think about this yet. We will probably display the same component but then a variation of it display.

Reply:
This is already implemented in the existing plans features components, as I suggested in other comments in this PR, could we have borrowed more from it?